src/odemis/driver/xt_client.py

```diff
@@ -2674,14 +2675,21 @@ def _acquire(self):
                 self._acq_wait_start()
                 logging.debug("Preparing acquisition")
                 while True:
-                    if self._acq_should_stop():
+                    logging.debug("Start acquiring an image")
+
+                    # The 0.2 timeout is needed to make sure it is worthy to start a new acquisition.
```

Comment:
I think it'd be worthy to indicate this is a hack. Something like "HACK: as cancellation doesn't work, to avoid acquiring an extra image at the end of an acquisition, we wait a little bit before starting the next frame. This gives enough time for the client after receiving the previous frame to decide that this is enough. Especially worthy when acquiring a single image."